src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -101,7 +107,13 @@ public async Task<IActionResult> List(ListContentsViewModel model, PagerParamete
                     break;
             }
 
-            if (contentTypeId != "")
+            if (model.Options.ContentsStatus == ContentsStatus.Owner)
+            {
+                var httpContext = _httpContextAccessor.HttpContext;
```

Comment:
That's why I suggest earlier to introduce `_httpContext` instead of `httpContextAccessor`